src/app/features/post/post.component.html

```diff
@@ -21,7 +21,12 @@ <h2>{{ post.excerpt }}</h2>
         </span>
         <div class="post__header-authors">
           @for (author of post.authors; track $index) {
-            <blog-author [author]="author" size="lg"></blog-author>
+            <a
+              [attr.data-role]="$any(author).url ? 'button' : ''"
+              [routerLink]="'/' + $any(author).url"
+              (click)="$event.stopPropagation()">
```

Comment:
Can you please double-check whether this `stopPropagation` is needed? I actually can't see the use of it here
```suggestion
              >
```

Reply:
You are right. no need. I needed it for `post-item.component.html`